test/html/expect/test_package+ml/Recent/index.html

```diff
@@ -30,22 +30,22 @@ <h1>
    </div>
    <dl>
     <dt class="spec type" id="type-variant">
-     <a href="#type-variant" class="anchor"></a><code><span class="keyword">type</span> variant</code><code> = </code>
-     <table class="variant">
+     <a href="#type-variant" class="anchor"></a><code><span class="keyword">type</span> variant = </code>
+     <table>
       <tbody>
        <tr id="type-variant.A" class="anchored">
-        <td class="def constructor">
-         <a href="#type-variant.A" class="anchor"></a><code>| </code><code><span class="constructor">A</span></code>
+        <td class="def variant constructor">
```

Comment:
> Class for type declarations have moved a bit (this probably requires changing the CSS, I will do that later).

Could you comment on why this has happened?

Reply:
I think it's better explained by looking at the representation of typedecls in the IR:
a typedecl is a `DocumentedSrc.t` and we have

```
  type 'a documented = {
    attrs : Class.t ;
    anchor : Url.Anchor.t option ;
    code : 'a ;
    doc : Block.t ;
  }
  type t = one list
  and one =
    | Code of only_code
    | Documented of Inline.t documented
    | Nested of t documented
```

The shape of the IR made it fairly natural to put annotations/classes on each line, instead on the global thing. 

Now that I think about it, it wouldn't actually be much effort at all to fix this. I'll do that.

We can also discuss the shape of the IR. I converged towards this one because it helped me produce the cleanest code, but it's of course not set in stone.

Reply:
Actually, it's more awkward than I though because of polymorphic variants. I propose to push that in a later PR. :)

Reply:
Ok. We can also change the CSS if that's the easier way.

Reply:
That was my plan.